File: HowGotEm/src/app/models/shoe-dto.ts
import { SizeDto } from "./size-dto";

export interface ShoeDto {
  model: string,
  skuCode: string,
  brand: string,
  color: string,
  sizes: SizeDto[],
  urlImg: string,
  urlImg2: string,
  urlImg3: string,
  bestSeller: boolean
}

File: howgotem_backend/src/main/java/it/alexpaganucci/howgotem/payloads/ShoeDto.java
package it.alexpaganucci.howgotem.payloads;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ShoeDto {

    private String model;
    private String skuCode;
    private String brand;
    private String color;
    private List<SizeDto> sizes;
    private String urlImg;
    private String urlImg2;
    private String urlImg3;
    private Boolean bestSeller;
    
}
